### DISCUSSION TITLE
Settings sidebar translations overflow in certain languages

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#15603] [Q&A] [answered]
german
![image](https://user-images.githubusercontent.com/43358824/141605419-97890efc-4704-4c08-ac4a-4c34b684eed8.png)
french
![image](https://user-images.githubusercontent.com/43358824/141605431-117e549e-362a-43f8-a602-099707030a11.png)
japanese
![image](https://user-images.githubusercontent.com/43358824/141605461-4a4131d1-4635-4198-a55b-d9fdbca2cf11.png)

and many other languages

**Comment by gagahpangeran** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15452
